### PR TITLE
OSS-RKE-version-update

### PR DESCRIPTION
**Pull request by bmckercher123**
### Update RKE version for OSS for 3.25 and next

### CODE REVIEW

File: calico/getting-started/kubernetes/rancher.mdx
Comment (ctauchen) [inline calico/getting-started/kubernetes/rancher.mdx:26]: Does this mean all 1.3.x releases? And not 1.4.x releases?

**Review by ctauchen — APPROVED** [review]
LGTM, pending SME approval

File: calico/getting-started/kubernetes/rancher.mdx
Comment (bmckercher123) [reply calico/getting-started/kubernetes/rancher.mdx:26]: Yes, it means just 1.3.x release. We have to revalidate all "next" releases with RKE.

**Review by mgleung — APPROVED** [review]

**Pull request merged**
